src/amuse/support/core.py

```diff
@@ -202,10 +202,10 @@ def string(self):
         return str(self)
         
     def isstring(self, x):
-        return isinstance(x,types.StringType)
+        return isinstance(x,bytes)
         
     def isnumber(self, x):
-        return isinstance(x,types.IntType) or isinstance(x,types.FloatType) 
+        return isinstance(x,int) or isinstance(x,float) 
         
         
 class OrderedDictionary(object):
```

Comment:
this one is tricky because the py23 converter has tried to maintain effectively python 2 behaviour, while the class mimicks a dict, so probably should use python3 dict conventions...

in any case nowadays there is a build-in ordered dict, unfortunately it behaves not exactly the same

Reply:
#489